### DISCUSSION TITLE
[Feature Request] Disabling comments

### DISCUSSION THREAD

**Discussion by crxssed7** [trakt/showly#608] [Ideas / Feature Requests]
Not sure if this is worth the implementation as it is very specific to me but maybe an option to disable the comments section on tv/episode/movie screens. I often end up wasting time reading comments and can't count how many times I've seen people not use the spoiler tag, so I'd rather just not have that section at all 😅

**Comment by michaldrabik** [comment]
Yeaahh I'm not sure I will go on with that. Comments are hidden in a separate sections so you should just not go in there if you dont want to. Adding another option in settings will not stop you I suppose ;)
